### PR TITLE
WIP: Add notes about how to join the conda-forge team

### PR DESCRIPTION
**Pull request by ericdill**
I pushed up what I had written before my kiddo woke up from her nap. Will update more as time permits :)

Other folks are welcome to push commits to this branch with changes

### CODE REVIEW

File: src/joining-the-team.rst [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+staged-recipes responsibilities
+===============================
+You are the welcoming committee for new recipes coming in to the conda-forge
+community! Please give new (and experienced) contributors a pleasant experience!
+Generally speaking your role is as follows:
+
+1. Keep up to date with the current best practices for conda packaging standards
+2. Provide recipe review which generally means making sure that the recipe
+   under review adheres to what we list on the :ref:`recipe`__ page.
+3. <Other people please chime in here with what the responsibilities are supposed to be>
```
Comment (CJ-Wright) [inline src/joining-the-team.rst]: Open issues as needed (both on staged and on the other flagship repos (smithy, webservices, etc)), especially when problems occur.

**Comment by jakirkham** [comment]
Thanks for doing this @ericdill! Generally seems fine to me.

File: src/joining-the-team.rst [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+rights to conda-forge/staged-recipes so that they can help us review and merge
+submissions more quickly!
+
+staged-recipes responsibilities
+===============================
+You are the welcoming committee for new recipes coming in to the conda-forge
+community! Please give new (and experienced) contributors a pleasant experience!
+Generally speaking your role is as follows:
+
+1. Keep up to date with the current best practices for conda packaging standards
```
Comment (CJ-Wright) [inline src/joining-the-team.rst]: We might consider putting out a changelog of best practices somewhere so people can be up to date.

File: src/joining-the-team.rst (hunk 1 above)
Comment (ericdill) [reply src/joining-the-team.rst]: @cj-wright can you push a commit with the suggested changes?

**Comment by ericdill** [comment]
Merging this in. We can iterate/add more as needed

File: src/joining-the-team.rst (hunk 2 above)
Comment (jakirkham) [reply src/joining-the-team.rst]: There's [the guidelines]( https://conda-forge.org/docs/guidelines.html ). We could also start versioning the docs. They have a version, but haven't tried to come up with a scheme yet.

Suggestions welcome. Feel free to put this in a new issue/PR of your choosing. :)

**Comment by jakirkham** [comment]
Thanks all. :)

**Pull request merged**